forester/src/utils.rs

```diff
@@ -0,0 +1,74 @@
+use crate::constants::SERVER_URL;
+use log::info;
+use solana_client::rpc_client::RpcClient;
+use solana_sdk::commitment_config::CommitmentConfig;
+use solana_sdk::pubkey::Pubkey;
+use std::process::Command;
+use sysinfo::{Signal, System};
+
+pub fn request_airdrop(payer_pubkey: &Pubkey) {
```

Comment:
didn't see where this is used, just saying since we cannot use airdrops on mainnet should try to not hardcode such logic that we need to clean up later in random places.

Reply:
if it's just used in tests we could move it to test-utils